### PR TITLE
3 small yet annoying bugs fixed

### PR DESCRIPTION
**Pull request by oazmon**

### CODE REVIEW

File: src/test/java/net/lightbody/bmp/proxy/ProxyServerTest.java [hunk 1]
```diff
@@ -62,6 +67,25 @@ public static DefaultHttpClient getNewHttpClient(int proxyPort) {
         }
     }
 
+	@Test
+	public void testHttpProxyOption() throws Exception {
+		ProxyServer instance = new ProxyServer(8082);
+		Map<String, String> options = new HashMap<String, String>();
+		options.put("httpProxy", "localhost:80801");
```
Comment (jekh) [inline src/test/java/net/lightbody/bmp/proxy/ProxyServerTest.java]: Typo? Port 80801 isn't valid.

File: src/test/java/net/lightbody/bmp/proxy/MailingListIssuesTest.java [hunk 2]
```diff
@@ -1,283 +0,0 @@
-package net.lightbody.bmp.proxy;
```
Comment (jekh) [inline src/test/java/net/lightbody/bmp/proxy/MailingListIssuesTest.java]: Did you accidentally rm this file?

File: src/test/java/net/lightbody/bmp/proxy/ProxyServerTest.java (hunk 1 above)
Comment (oazmon) [inline src/test/java/net/lightbody/bmp/proxy/ProxyServerTest.java]: Doesn't matter for this test. But fixed it.

**Comment by jekh** [comment]
@oazmon -- could you recreate this PR against the 2.0 branch, with the change above?

**Pull request closed without merging**